runelite-api/src/main/java/net/runelite/api/Varbits.java

```diff
@@ -421,7 +421,17 @@
 	CURRENT_BANK_TAB(4150),
 
 	WORLDHOPPER_FAVROITE_1(4597),
-	WORLDHOPPER_FAVROITE_2(4598);
+	WORLDHOPPER_FAVROITE_2(4598),
+
+	/**
+	 * Vengeance is active
+	 */
+	VENGEANCE_ACTIVE(2450),
+
+	/**
+	 * Spell cooldowns
+	 */
+	VENGEANCE_VENGEANCE_OTHER_COOLDOWN(2451);
```

Comment:
If the cooldowns are shared why is this called `VENGEANCE_VENGEANCE_OTHER_COOLDOWN`?

Reply:
Because this affects both Vengeance and Vengeance Other? What are you expecting it to be called?

Reply:
As Nightfire said. I could name it `VENGEANCE_AND_VENGEANCE_OTHER_COOLDOWN`to make it even more clear, but that feels a bit excessive?